crates/metrics/src/txpool_metrics.rs

```diff
@@ -1,24 +1,89 @@
 use crate::global_registry;
-use prometheus_client::metrics::histogram::Histogram;
+use prometheus_client::metrics::{
+    gauge::Gauge,
+    histogram::Histogram,
+};
 use std::sync::OnceLock;
 
 pub struct TxPoolMetrics {
-    pub tx_size_histogram: Histogram,
+    /// Size of transactions in the txpool in bytes
+    pub tx_size: Histogram,
+    pub number_of_transactions: Gauge,
+    pub number_of_transactions_pending_verification: Gauge,
+    pub number_of_executable_transactions: Gauge,
+    /// Time of transactions in the txpool in seconds
+    pub transaction_time_in_txpool_secs: Histogram,
+    /// How long it took for the selection algorithm to select transactions
+    pub select_transaction_time_nanoseconds: Histogram,
 }
 
 impl Default for TxPoolMetrics {
     fn default() -> Self {
-        let tx_sizes = Vec::new();
+        let tx_sizes = Vec::new(); // TODO: What values for tx_sizes?
+        let tx_size = Histogram::new(tx_sizes.into_iter());
+        let transaction_time_in_txpool_secs = Histogram::new(
+            vec![
+                1.0, 2.0, 3.0, 4.0, 5.0, 10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 120.0,
+                240.0, 480.0, 960.0, 1920.0, 3840.0,
+            ]
+            .into_iter(),
+        );
+        let select_transaction_time_nanoseconds = Histogram::new(
+            vec![
+                10.0, 20.0, 30.0, 40.0, 50.0, 100.0, 200.0, 500.0, 1000.0, 2000.0,
+                5000.0, 10000.0, 50000.0, 100000.0, 500000.0,
+            ]
+            .into_iter(),
```

Comment:
We should move these to `buckets.rs`.

Reply:
I moved this [here](https://github.com/FuelLabs/fuel-core/pull/2321/commits/0fa3a3946770d318a8cfe9b1dc0de86a2648489c) and also created a bucket for transaction sizes [here](https://github.com/FuelLabs/fuel-core/pull/2321/commits/5c6c3f0effcd94b3b44ec2fbc83f85a26bb51cdd).